include/ideep/operators/conv.hpp

```diff
@@ -132,8 +132,9 @@ struct conv_deconv_utils {
         // Here we need to recalculate the scale of sum.
         // When fused with sum, dst_scales_in is the final output tensor's scale.
         // dst.scale is the scale of sum tensor.
-        float sum_scale =
-            dst_scales_in[0] / (dst.has_scale() ? dst.get_scale()[0] : 1.0f);
+
+        float sum_scale = 1.0 / (dst.has_scale() ? dst.get_scale()[0] : 1.0f);
```

Comment:
Does linear need this fix too?
https://github.com/intel/ideep/blob/380142290c0be4cc6170819aff1e498a01971d17/include/ideep/operators/matmul.hpp#L991

Can you please check all the related places that are impacted and fix them too?

Reply:
QLinear add fusion are not enabled, so I think that's qconv add is the only position to be fixed.

Reply:
But is it an issue? If so, we should fix it?

Reply:
Thanks and updated. Also audit the ideep code, the other place used with fused sum are not for quantization.